ExecutionHelper/.github/CODEOWNERS

```diff
@@ -0,0 +1,10 @@
+# This CODEOWNERS file sets the individuals responsible for code in the zksync-hardhat-template repository.
+
+# These users are the default owners for everything in the repo.
+# They will be requested for review when someone opens a pull request.
+* @matter-labs/devxp
```

Comment:
We should fix this. Do we have a value for lambdaclass?

Reply:
Maybe @ilitteri can answer this?

Reply:
i did git init to some configuration, that create the .github folder with matter configuration, just delete it.